pinot-spi/pom.xml

```diff
@@ -36,6 +36,38 @@
 
   <build>
     <plugins>
+      <plugin>
+        <groupId>org.apache.maven.plugins</groupId>
+        <artifactId>maven-dependency-plugin</artifactId>
+        <version>3.7.1</version>
+        <executions>
+          <execution>
+            <id>copy-pinot-plugins</id>
+            <phase>process-test-resources</phase>
+            <goals>
+              <goal>copy</goal>
+            </goals>
+            <configuration>
+              <artifactItems>
+                <artifactItem>
+                  <groupId>org.apache.pinot</groupId>
+                  <artifactId>pinot-dropwizard</artifactId>
+                  <version>0.10.0</version>
```

Comment:
nit: should we use here $project.version instead?

Reply:
I prefer it like this, because it makes the test more solid. In case pinot-dropwizard would change its content or would be removed, this test would suddenly fail. So that means that the developer working on pinot-dropwizard breaks tests of pinot-spi!?!? That is so unexpected (the otherway around could make sense: changes in pinot-spi could break plugins)

Reply:
makes sense.